### PR TITLE
Added ECDSA documentation

### PR DESCRIPTION
**Pull request by joepegler**
Ecdsa-documentation

### CODE REVIEW

**Comment by himanshugarg06** [comment]
@VGabriel45 can you review as well, think it was updated by you recently. https://github.com/bcnmy/docs/commit/40fc6a9f6e60abd16a53cbfa165799e9d3fe2d09#diff-4333a7b11c2033a0fdc7d906bbcf403e42df478b80edc972d7232fbd992982b4

we should have the basic integration setup.

File: docs/Modules/ecdsa.mdx [hunk 1]
```diff
@@ -38,6 +38,49 @@ One of the most popular modules, it bridges the ease of traditional web logins w
...
+
+### Initializing ECDSA Module 🌟
+
+There is no need to do anything in order to use ECDSAOwnershipValidationModule, this module is set by default on every smart account.
+
+### Usage
+
+| Key                                                                                                              | Description                                                                                                                           |
+| ---------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| [signer](https://bcnmy.github.io/biconomy-client-sdk/packages/account/docs/interfaces/SmartAccountSigner.html)   | This signer will be used for signing userOps for any transactions you build. Will accept ethers.JsonRpcSigner as well as a viemWallet |
```
Comment (himanshugarg06) [inline docs/Modules/ecdsa.mdx:51]: these links are not accessible.

File: docs/Modules/ecdsa.mdx [hunk 2]
```diff
@@ -38,6 +38,49 @@ One of the most popular modules, it bridges the ease of traditional web logins w
...
 
+## SDK Guide 📘
+
+### Initializing ECDSA Module 🌟
+
+There is no need to do anything in order to use ECDSAOwnershipValidationModule, this module is set by default on every smart account.
+
+### Usage
+
+| Key                                                                                                              | Description                                                                                                                           |
```
Comment (himanshugarg06) [inline docs/Modules/ecdsa.mdx:49]: are these both required param to create the instance?

**Review by himanshugarg06 — CHANGES_REQUESTED** [review]

File: docs/Modules/ecdsa.mdx (hunk 1 above)
Comment (joepegler) [reply docs/Modules/ecdsa.mdx:51]: They will be once this is merged: https://github.com/bcnmy/biconomy-client-sdk/pull/418

File: docs/Modules/ecdsa.mdx (hunk 2 above)
Comment (joepegler) [reply docs/Modules/ecdsa.mdx:49]: No they're not required. Signer is required but config is optional

**Review by himanshugarg06 — APPROVED** [review]

**Pull request merged**